types/mysql-query-util/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for mysql-query-util 0.2
+// Project: https://github.com/uchennaemeruche/mysql-util#readme
+// Definitions by: Uchenna Emeruche <https://github.com/uchennaemeruche>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+export as namespace myLib;
```

Comment:
This doesnt seem to show usage globally from https://www.npmjs.com/package/mysql-query-util. 
If this is correct you need to add global usage test
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.